### ISSUE TITLE
What is wrong with my pickit?

### ISSUE THREAD

**Issue by LolO-o** [kolton/d2bot-with-kolbot#2393]
I am trying to get a 3 raise skeleton 3 skeleton mastery head(and wand) but kolbot spews out the following: Unknown property: skillraiseskeleton 

here is my pickit
[name] == preservedhead && [quality] == superior # [skillraiseskeleton]+[skillskeletonmastery] >= 6 )
[name] == preservedhead && [quality] == superior # [skillraiseskeleton]+[skillskeletonmastery] >= 5 )
[name] == zombiehead && [quality] == superior # [skillraiseskeleton]+[skillskeletonmastery] >= 6 )
[name] == zombiehead && [quality] == superior # [skillraiseskeleton]+[skillskeletonmastery] >= 5 )
[type] == wand && [skillraiseskeleton]+[skillskeletonmastery] == 6

completely puzzled on what im doing wrong

**Comment by DarkHorseDre** [comment]
I've seen this type of thing but not sure whassup - it does exist in NTItemAlias.dbl at least. You're gonna need more knowledgeable advice here..

in the meantime, the 3rd and 4th line have closing brackets ")" but no opening

**Comment by LolO-o** [comment]
Yeah i edited the brackets..my bad...but still no bueno and it seems like a problem with both raise skeleton and skeleton mastery(i tried switching them around). I know that skeletal mages use skillskeletalmage instead of skillraiseskeletalmage but "skillraiseskeleton" are is "skillraiseskeleton" according to NTIemAlias.dbl

**Comment by DarkHorseDre** [comment]
yeah, this item translation stuffs isnt 100% - I dont understand it though.

that said, I loaded your lines and it's only the wand line I'm getting an error for - the rest are fine.

**Comment by Machoclown** [comment]
IMHO it should be:
[type] == wand # [skillraiseskeleton]+[skillskeletonmastery] == 6

**Comment by DarkHorseDre** [comment]
ha! good catch @Machoclown - don't know how i didnt see that - those # are key!
